README.md

```diff
@@ -21,16 +21,9 @@ dependencies {
 [![Version Notifications](https://www.bintray.com/docs/images/bintray_badge_bw.png)](https://bintray.com/duolingo/maven/RtlViewPager?source=watch)
 
 ## License
-> Copyright 2016 Duolingo
 
-> Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+Please read our [contributing guidelines](CONTRIBUTING.md) prior to submitting a Pull Request.
```

Comment:
To wordsmith a little:  Let's leave out "prior to submitting a Pull Request" since the guidelines don't say to do anything other than submit a pull request.